.github/workflows/ubuntu_22.yml

```diff
@@ -88,6 +88,111 @@ jobs:
       event-name: ${{ github.event_name }}
       os: 'ubuntu_22_04'
 
+  Build_with_Faster_Build_Option:
+    name: Build with FASTER_BUILD=ON
```

Comment:
if we want to test this option, we can enable it in CC Collect Job and avoid new jobs.